torch/nn/_intrinsic/qat/modules/conv_fused.py

```diff
@@ -80,13 +80,17 @@ def reset_parameters(self):
             self.reset_bn_parameters()
 
     def enable_fake_quant(self):
-        self.observer.enable()
-        self.weight_fake_quant.enable()
+        self.observer.enable_fake_quant()
```

Comment:
why do you need this? I guess the .apply should work recursively?

Reply:
Good catch, that's not needed, removed it.